surf_trashbin/js/app.js

```diff
@@ -27,7 +27,12 @@ $(document).ready(function() {
 						return;
 					}
 					this._initialized = true;
-					asdasdasd(groupName);
+
+					let headerNameContainer = contents[i].children[5].children[0].children[0].children[0].children[0];
```

Comment:
This feels very brittle especially the magic value `5` there, is there another way to get the header name container? Does it not have an id or a class or something?

Reply:
It can be done better. At the time I was under time pressure and went for the quickest approach. It is brittle but I also think it won't break unless there are major changes in the OC core UI